tests/organization_subqueries.spec.js

```diff
@@ -5,29 +5,104 @@ describe('organization resolvers', () => {
   test('organization-subqueries', async () => {
     const response = await axios.post(URL, {
       query: `
```

Comment:
How is your query running, it should throw an error. Currently nesting level in graph is allowed upto 3, more than that should throw an error

Reply:
Yeah, here the nesting level is 3 only.